src/ckb/channel.rs

```diff
@@ -2519,7 +2519,10 @@ impl ChannelActorState {
             }
         };
         let tx_builder = TransactionBuilder::default()
-            .cell_deps(get_cell_deps_by_contracts(vec![Contract::CommitmentLock]))
+            .cell_deps(get_cell_deps_by_contracts(vec![
+                Contract::CommitmentLock,
```

Comment:
I made an error here previously. We actually need only the `FundingLock` because we are building a transaction to consume the funding cell. `CommitmentLock` is not required.